<!DOCTYPE html>
<html>

<head>
  <link href="style.css" rel="stylesheet" type="text/css" />

  <script src="https://kit.fontawesome.com/5f10c058c5.js" crossorigin="anonymous"></script>
</head>
<div class="scrollmenu">
  <a href="index.html" accesskey="h" title="Human Pangenome (Alt + H)">
    <img src="img/Logo.png" alt="logo" height="40" class="logo"> </a>
  <div class="topnav-right">
    <div class="dropdown">
      <a href="aboutus.html" class="dropbtn">About Us</a>
      <div class="dropdown-content">
        <a href="about/consortium.html">Consortium Organization</a><br>
        <a href="about/advisory.html">Advisory Board</a><br>
        <a href="about/partners.html">Partners</a><br>
        <a href="about/assoc.html">Associate Members</a><br>
        <a href="about/ga4gh.html">GA4GH Commitments</a><br>
        <a href="about/covid.html">COVID-19 Response</a><br>
        <a href="about/rsr.html">Response to Systematic Racism </a>
      </div>
    </div>
    <a href="data.html">Data</a>
    <a href="tech.html">Technology</a>
    <a href="sample.html">Samples</a>
    <a href="publ.html">Publications</a>
    <a href="contact.html">Contact</a>
    <a href="http://wiki.humanpangenome.org/" target="_blank"><button id="wiki"> Wiki Login </button></a>
  </div>
</div>
<style>
  tbody>tr>:nth-child(1) {
    width: 30%;
  }

  tbody>tr>:nth-child(2) {
    width: 30%;
  }

  tbody>tr>:nth-child(3) {
    width: 20%;
  }

  tbody>tr>:nth-child(4) {
    width: 20%;
  }

  td {
    padding-block: 5px 10px;
  }

  td {
    padding-right: 5px;
  }

  td {
    text-align: center;
  }

  tbody>tr>:nth-child(1) {
    text-align: left;
  }
</style>
<main>
  <h1>Publications</h1>
  <p>There are several HPRC publications in development at this time. Please check back for updates and follow us on
    Twitter <a href="https://twitter.com/HumanPangenome" target="_blank">@HumanPangenome</a>. Below is a selection of
    publications from the HPRC membership.</p>
  <table>
    <thead>
      <tr>
        <th>Title</th>
        <th>Authors</th>
        <th>DOI</th>
        <th>Date</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>Pangenome-based genome inference allows efficient and accurate genotyping across a wide spectrum of variant
          classes.</td>
        <td>Ebler J, Ebert P, Clarke WE, Rausch T, Audano PA, Houwaart T, Mao Y, Korbel JO, Eichler EE, Zody MC, Dilthey
          AT, Marschall T.</td>
        <td><a href="https://www.nature.com/articles/s41588-022-01043-w" target="_blank">10.1038/s41588-022- 01043-w</a>
        </td>
        <td>2022-04-11</td>
      </tr>
      <tr>
        <td>Concerted modification of nucleotides at functional centers of the ribosome revealed by single-molecule RNA
          modification profiling.</td>
        <td>Bailey AD, Talkish J, Ding H, Igel H, Duran A, Mantripragada S, Paten B, Ares M.Elife</td>
        <td><a href="https://elifesciences.org/articles/76562" target="_blank">10.7554/eLife.76562</a></td>
        <td>2022-04-06</td>
      </tr>
      <tr>
        <td>Segmental duplications and their variation in a complete human genome.</td>
        <td>Vollger MR, Guitart X, Dishuck PC, et al.</td>
        <td><a href="https://www.science.org/doi/10.1126/science.abj6965" target="_blank">10.1126/science.abj69 65</a>
        </td>
        <td>2022-04-01</td>
      </tr>
      <tr>
        <td>StainedGlass: Interactive visualization of massive tandem repeat structures with identity heatmaps.</td>
        <td>Vollger MR, Kerpedjiev P, Phillippy AM, Eichler EE.</td>
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/35020798/" target="_blank">10.1093/bioinformatics /btac018</a></td>
        <td>2022-01-10</td>
      </tr>
      <tr>
        <td>The UCSC Genome Browser database: 2022 update.</td>
        <td>Lee BT, Barber GP, Benet-Pag√®s A, et al.</td>
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/34718705/" target="_blank">10.1093/nar/gkab959</a></td>
        <td>2022-01-07</td>
      </tr>
      <tr>
        <td>Profiling variable-number tandem repeat variation across populations using repeat-pangenome graphs</td>
        <td>Tsung-Yu Lu, Human Genome Structural Variation Consortium, Mark Chaisson.</td>
        <td><a href="https://doi.org/10.1038/s41467-021-24378-0" target="_blank">10.1038/s41467-021- 24378-0</a></td>
        <td>2021-07-12</td>
      </tr>
      <tr>
        <td>Targeted long-read sequencing identifies missing disease-causing variation</td>
        <td>Miller, D., Sulovari, A., Wang, T., et al.</td>
        <td><a href="https://doi.org/10.1016/j.ajhg.2021.06.006" target="_blank">10.1016/j.ajhg.2021.06 .006</a></td>
        <td>2021-07-02</td>
      </tr>
      <tr>
        <td>lra: A long read aligner for sequences and contigs</td>
        <td>Jingwen Ren, Mark Chaisson.</td>
        <td><a href="https://doi.org/10.1371/journal.pcbi.1009078" target="_blank">10.1371/journal.pcbi.10 09078</a>
        </td>
        <td>2021-06-21</td>
      </tr>
      <tr>
        <td>The qBED track: a novel genome browser visualization for point processes</td>
        <td>Moudgil, A., Li, D., Hsu, S., et al.</td>
        <td><a href="https://doi.org/10.1093/bioinformatics/btaa771" target="_blank">10.1093/bioinformatics /btaa771</a>
        </td>
        <td>2021-05-23</td>
      </tr>
      <tr>
        <td>Common DNA methylation dynamics in endometriod adenocarcinoma and glioblastoma suggest universal epigenomic
          alterations in tumorigenesis</td>
        <td>Karlow, J., Miao, B., Xing, X., et al.</td>
        <td><a href="https://doi.org/10.1038/s42003-021-02094-1" target="_blank">10.1038/s42003-021- 02094-1</a></td>
        <td>2021-05-21</td>
      </tr>
      <tr>
        <td>A high-quality bonobo genome refines the analysis of hominid evolution</td>
        <td>Mao, Y., Catacchio, C., Hillier, L., et al.</td>
        <td><a href="https://doi.org/10.1038/s41586-021-03519-x" target="_blank">10.1038/s41586-021- 03519-x</a></td>
        <td>2021-05-05</td>
      </tr>
      <tr>
        <td>The Need for a Human Pangenome Reference Sequence</td>
        <td>Miga, K., Wang, T</td>
        <td><a href="https://doi.org/10.1146/annurev-genom-120120-081921" target="_blank">10.1146/annurev-gen
            om-120120-081921</a></td>
        <td>2021-04-30</td>
      </tr>
      <tr>
        <td>Human placental cytotrophoblast epigenome dynamics over gestation and alterations in placental disease</td>
        <td>Zhang, B., Kim, B., Elliot, G., et al.</td>
        <td><a href="https://doi.org/10.1016/j.devcel.2021.04.001" target="_blank">10.1016/j.devcel.2021 .04.001</a>
        </td>
        <td>2021-04-20</td>
      </tr>
      <tr>
        <td>Profiling SARS-CoV-2 mutation fingerprints that range from the viral pangenome to individual infection
          quasispecies</td>
        <td>Lau, B., Pavlichin, D., Hooker, A., et al.</td>
        <td><a href="https://doi.org/10.1186/s13073-021-00882-2" target="_blank">10.1186/s13073-021- 00882-2</a></td>
        <td>2021-04-19</td>
      </tr>
      <tr>
        <td>The structure, function and evolution of a complete human chromosome 8</td>
        <td>Logsdon, G., Vollger, M., Hsieh, P., et al.</td>
        <td><a href="https://doi.org/10.1038/s41586-021-03420-7" target="_blank">10.1038/s41586-021- 03420-7</a></td>
        <td>2021-04-07</td>
      </tr>
      <tr>
        <td>Haplotype-resolved diverse human genomes and integrated analysis of structural variation</td>
        <td>Ebert, P., Audano, P., Zhu, Q., et al.</td>
        <td><a href="https://doi.org/10.1126/science.abf7117" target="_blank">10.1126/science.abf7 117</a></td>
        <td>2021-02-25</td>
      </tr>
      <tr>
        <td>Haplotype-resolved de novo assembly using phased assembly graphs with hifiasm</td>
        <td>Cheng, H., Concepcion, G., Feng, X., et al.</td>
        <td><a href="https://doi.org/10.1038/s41592-020-01056-5" target="_blank">10.1038/s41592-020- 01056-5</a></td>
        <td>2021-02-01</td>
      </tr>
      <tr>
        <td>Fully phased human genome assembly without parental data using single-cell strand sequencing and long reads
        </td>
        <td>Porubsky, D., et al.</td>
        <td><a href="https://doi.org/10.1038/s41587-020-0719-5" target="_blank">10.1038/s41587-020- 0719-5</a></td>
        <td>2020-12-07</td>
      </tr>
      <tr>
        <td>Chromosome-scale, haplotype-resolved assembly of human genomes</td>
        <td>Garg, S., Fungtammasan, A., Carroll, A., et al.</td>
        <td><a href="https://doi.org/10.1038/s41587-020-0711-0" target="_blank">10.1038/s41587-020- 0711-0</a></td>
        <td>2020-12-07</td>
      </tr>
      <tr>
        <td>A map of cis-regulatory elements and 3D genome structures in zebrafish</td>
        <td>Yang, H., Luan, Y., Liu, T., et al.</td>
        <td><a href="https://doi.org/10.1038/s41586-020-2962-9" target="_blank">10.1038/s41586-020- 2962-9</a></td>
        <td>2020-11-25</td>
      </tr>
      <tr>
        <td>The design and construction of reference pangenome graphs with minigraph</td>
        <td>Li, H., Feng, X., Chu, C.</td>
        <td><a href="https://doi.org/10.1186/s13059-020-02168-z" target="_blank">10.1186/s13059-020- 02168-z</a></td>
        <td>2020-10-16</td>
      </tr>
      <tr>
        <td>Cell-type-specific 3D epigenomes in the developing human cortex</td>
        <td>Song, M., Pebworth, M., Yang, X., et al.</td>
        <td><a href="https://doi.org/10.1038/s41586-020-2825-4" target="_blank">10.1038/s41586-020- 2825-4</a></td>
        <td>2020-10-14</td>
      </tr>
      <tr>
        <td>Tissue-specific usage of transposable element-derived promoters in mouse development</td>
        <td>Miao, B., Fa, S., Lyu, C., et al.</td>
        <td><a href="https://doi.org/10.1186/s13059-020-02164-3" target="_blank">10.1186/s13059-020- 02164-3</a></td>
        <td>2020-09-28</td>
      </tr>
      <tr>
        <td>Telomere-to-telomere assembly of a complete human X chromosome</td>
        <td>Miga, K., Koren, S., Rhie, A. et al.</td>
        <td><a href="https://doi.org/10.1038/s41586-020-2547-7" target="_blank">10.1038/s41586-020- 2547-7</a></td>
        <td>2020-07-14</td>
      </tr>
      <tr>
        <td>Epigenomics and genotype-phenotype association analyses reveal conserved genetic architecture of complex
          traits in cattle and human</td>
        <td>Liu, S., Yu, Y., Zhang, S., et al.</td>
        <td><a href="https://doi.org/10.1186/s12915-020-00792-6" target="_blank">10.1186/s12915-020- 00792-6</a></td>
        <td>2020-07-03</td>
      </tr>
      <tr>
        <td>The design and construction of reference pangenome graphs</td>
        <td>Li, H., Feng, X., Chu, C.</td>
        <td><a href="https://arxiv.org/abs/2003.06079" target="_blank">arXiv:2003.06079</a></td>
        <td>2020-03-13</td>
      </tr>
      <tr>
        <td>Genotyping structural variants in pangenome graphs using the vg toolkit</td>
        <td>Hickey, G., Heller, D., Monlong, J., et al.</td>
        <td><a href="https://doi.org/10.1186/s13059-020-1941-7" target="_blank">10.1186/s13059-020- 1941-7</a></td>
        <td>2020-02-12</td>
      </tr>
      <tr>
        <td>A fully phased accurate assembly of an individual human genome</td>
        <td>Porubsky, D., Ebert, P., Audano, P., et al.</td>
        <td><a href="https://doi.org/10.1101/855049" target="_blank">10.1101/855049</a></td>
        <td>2019-11-26</td>
      </tr>
      <tr>
        <td>Efficient de novo assembly of eleven human genomes using PromethION sequencing and a novel nanopore toolkit
        </td>
        <td>Shafin, K., Pesout, T., Lorig-Roach, R., et al.</td>
        <td><a href="https://doi.org/10.1101/715722" target="_blank">10.1101/715722</a></td>
        <td>2019-07-26</td>
      </tr>
      <tr>
        <td>Progressive alignment with Cactus: a multiple-genome aligner for the thousand-genome era</td>
        <td>Armstrong, J., Hickey, G., Diekhans, M., et al.</td>
        <td><a href="https://doi.org/10.1101/730531" target="_blank">10.1101/730531</a></td>
        <td>2019-10-09</td>
      </tr>
    </tbody>
  </table>
</main>
<div class="icon">
  <br>
  <a href="https://twitter.com/HumanPangenome" target="_blank"><i style='font-size:48px;color:rgb(0, 172, 238)'
      class='fab'>&#xf081;</i></a>
  <a href="https://youtube.com" target="_blank"><i style='font-size:48px;color:rgb(255, 0, 0)' class='fab'>&#xf431;</i></a>
  <a href="https://github.com" target="_blank"><i style='font-size:48px;color:rgb(110, 84, 148)' class='fab'>&#xf092;</i></a>
  <br><br>
</div>
<div class="footer-basic">
  <footer>
    <div class="social">
      <p class="hprc">HPRC</p>
      <ul class="list-inline">
        <li class="list-inline-item"><a href="index.html"><i class="fa-solid fa-house"></i> Home</a></li>
        <li class="list-inline-item"><a href="aboutus.html"><i class="fa-solid fa-circle-info"></i> About</a></li>
        <li class="list-inline-item"><a href="contact.html"><i class="fa-solid fa-envelope"></i> Contact</a></li>
      </ul>
      <p class="copyright">¬© 2022 Human Pangenome Reference Consortium</p>
    </div>
  </footer>
</div>

</html>